hera_mc/cm_utils.py

```diff
@@ -137,7 +164,10 @@ def get_astropytime(_date, _time=0):
         return_date = _date
     elif _date is None or _date is False:
         return_date = _date
-    else:
+    elif isinstance(_date, (int, long, float)):
+        if int(_date) > 1000000000:
+            return Time(_date, format='gps')
```

Comment:
If the input is 56646.6, this logic will lead to a return value of None, and not an exception. Is that the best behavior?